témata-dizertačních-prací/draft/index.html

```diff
@@ -169,6 +169,7 @@ <h4>Název</h4>
                             Příklad
                         </dt>
                         <dd>
+                            Více vícejazyčných textů: např. v JSON: <code>{"cs": "Platforma pro exploraci Linked Data", "en": "Platform for Linked Data Exploration"}</code>
```

Comment:
@martinvita Nerozumim, proč by jedna práce měla mít více názvů v jednom jazyku. To by byl v rámci OFN unikát. Jsi si tím jistý? Pak tedy nesedí ani ten příklad v JSONu.